class/defaults.yml

```diff
@@ -1,3 +1,8 @@
 parameters:
   prometheus_pushgateway:
     namespace: syn-prometheus-pushgateway
+    prometheus_rule_labels:
+      prometheus: main
+    prometheus_namespace: prometheus
```

Comment:
Usually the `PrometheusRule` CRs should be deployed in the same namespace as the actual app, so maybe default to `${prometheus_pushgateway:namespace}`?